### PR TITLE
Added pylint github action & also fixed syntactic error in subtract function

### PR DESCRIPTION
**Pull request by rish-singhal**
Hi, 
Have added pylint github action (needs some testing -- see if someone can help me with that, it would be great).
Also, fixed syntactic error in subtract function

### CODE REVIEW

File: .github/workflows/pylint.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Python Code Quality and Lint
+on:
+  push:
+    branches: [ develop ]
```
Comment (AllanRegush) [inline .github/workflows/pylint.yml]: we only have the `main` branch

File: .github/workflows/pylint.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  pylint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: ricardochaves/python-lint@v1.3.0
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (AllanRegush) [inline .github/workflows/pylint.yml]: I think we need a newline here :nerd_face:

**Review by AllanRegush — COMMENTED** [review]
Looks Good :+1:  Just some minor adjustments :nerd_face:

File: .github/workflows/pylint.yml (hunk 1 above)
Comment (rish-singhal) [reply .github/workflows/pylint.yml]: done updated

File: .github/workflows/pylint.yml (hunk 2 above)
Comment (rish-singhal) [reply .github/workflows/pylint.yml]: done! ... also I have seen this kind of practice before, but don't know the motive behind it. Can you point out why we need a newline? Thanks! :)

**Comment by rish-singhal** [comment]
added changes!

**Review by AllanRegush — APPROVED** [review]
Looks Good 👍 Thank you

**Pull request merged**